tests/test_boolean.py

```diff
@@ -324,217 +195,419 @@ def test_parse_side_by_side_symbols_with_parens_raise_exception(self):
-        self.assertEqual(algebra.FALSE.simplify(), algebra.FALSE)
+
+        assert algebra.TRUE.simplify() == algebra.TRUE
+        assert algebra.FALSE.simplify() == algebra.FALSE
 
     def test_dual(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.dual, algebra.FALSE)
-        self.assertEqual(algebra.FALSE.dual, algebra.TRUE)
+
+        assert algebra.TRUE.dual ==  algebra.FALSE
+        assert algebra.FALSE.dual ==  algebra.TRUE
 
     def test_equality(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE, algebra.TRUE)
-        self.assertEqual(algebra.FALSE, algebra.FALSE)
-        self.assertNotEqual(algebra.TRUE, algebra.FALSE)
+
+        assert algebra.TRUE == algebra.TRUE
+        assert algebra.FALSE == algebra.FALSE
+
+        assert     algebra.TRUE != algebra.FALSE
+        assert not algebra.TRUE == algebra.FALSE
 
     def test_order(self):
         algebra = BooleanAlgebra()
-        self.assertTrue(algebra.FALSE < algebra.TRUE)
-        self.assertTrue(algebra.TRUE > algebra.FALSE)
+
+        assert algebra.FALSE < algebra.TRUE
+        assert algebra.TRUE > algebra.FALSE
 
     def test_printing(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(str(algebra.TRUE), '1')
-        self.assertEqual(str(algebra.FALSE), '0')
-        self.assertEqual(repr(algebra.TRUE), 'TRUE')
-        self.assertEqual(repr(algebra.FALSE), 'FALSE')
 
+        assert str(algebra.TRUE) == '1'
+        assert str(algebra.FALSE) == '0'
 
-class SymbolTestCase(unittest.TestCase):
+        assert repr(algebra.TRUE) == 'TRUE'
+        assert repr(algebra.FALSE) == 'FALSE'
 
-    def test_init(self):
-        Symbol(1)
-        Symbol('a')
-        Symbol(None)
-        Symbol(sum)
-        Symbol((1, 2, 3))
-        Symbol([1, 2])
+class TestSymbolCase:
 
     def test_isliteral(self):
-        self.assertTrue(Symbol(1).isliteral is True)
+        assert Symbol(1).isliteral is True
 
     def test_literals(self):
         l1 = Symbol(1)
         l2 = Symbol(1)
-        self.assertTrue(l1 in l1.literals)
-        self.assertTrue(l1 in l2.literals)
-        self.assertTrue(l2 in l1.literals)
-        self.assertTrue(l2 in l2.literals)
-        self.assertRaises(AttributeError, setattr, l1, 'literals', 1)
+
+        assert l1 in l1.literals
+        assert l1 in l2.literals
+        assert l2 in l1.literals
+        assert l2 in l2.literals
+
+        for symbol in [l1, l2]:
+            with pytest.raises(AttributeError):
+                symbol.setattr('literals', 1)
 
     def test_literalize(self):
         s = Symbol(1)
-        self.assertEqual(s.literalize(), s)
+
+        assert s.literalize() == s
 
     def test_simplify(self):
         s = Symbol(1)
-        self.assertEqual(s.simplify(), s)
 
-    def test_equal_symbols(self):
+        assert s.simplify() == s
+
+    def test_symbols_eq_0(self):
+        algebra = BooleanAlgebra()
+
```

Comment:
Actually, not really. This shows three parts of the unit test: setup, run and assertion. But can make it three lines, no spaces if you want.